include/revng/EarlyFunctionAnalysis/ApplyRegisterStateDeductions.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include "revng/ADT/SmallMap.h"
+#include "revng/Model/ABI.h"
+#include "revng/Model/Register.h"
+
+namespace abi {
+
+namespace RegisterState {
+
+using namespace model::RegisterState;
+
+} // namespace RegisterState
```

Comment:
```suggestion
namespace abi::RegisterState  = model::RegisterState;
```
?

But why duplicate?

Reply:
Wait, I definitely tried to do that and the compiler complained (?). I'm probably mixing stuff up, sorry about that.

If the shorter form works, I prefer it as well.

Reply:
OK, but why not go with `model::RegisterState` directly?

Reply:
Actually, now that I think about it, giving it an alias will only lead to extra confusion. I'm sorry about this.

Reply:
It's gone.